uint8 command
uint8 ID
uint8 direction
float32 vel
float32 acc
int32 maxDistance
